taskcluster/adhoc_taskgraph/static_task_vars.py

```diff
@@ -0,0 +1,17 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+STATIC_VARS_BY_PRODUCT = {
```

Comment:
Awesome :)
The only comment I have is `STATIC_VARS_BY_PRODUCT` sounds like it'll apply for all platforms, and these are mac-specific. Maybe `MAC_STATIC_VARS_BY_PRODUCT`?

Also, do we need to adjust the iscript schema/logic at all for these?

Reply:
(have we verified a manual `taskgraph` run to make sure mac- and non-mask tasks look right?)